### PR TITLE
docs(samples): add policyTagManager samples

### PR DESCRIPTION
**Pull request by steffnay**
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

 🦕

### CODE REVIEW

File: samples/policyTagManager/deleteTaxonomy.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  // [START data_catalog_ptm_delete_taxonomy]
+  // Import the Google Cloud client library.
+  const {PolicyTagManagerClient} = require('@google-cloud/datacatalog').v1;
+  const policyTagManager = new PolicyTagManagerClient();
+
+  async function deleteTaxonomy() {
+    /**
+     * TODO(developer): Uncomment the following line before running the sample.
+     */
+    // const taxonomyName = 'my display name'; // Existing taxonomy.
```
Comment (shollyman) [inline samples/policyTagManager/deleteTaxonomy.js:27]: I believe the name arg should be for the qualified name, not the display name (e.g. projects//locations/us/taxonomies/1234)

**Review by shollyman — APPROVED** [review]
Mostly just comment nits, modulo one optional field as well.

**Comment by tmatsuo** [comment]
FYI, Justin's merge commits make our CLA bot unhappy. You can omit them then force push to resolve it.

**Comment by steffnay** [comment]
closing in favor of https://github.com/googleapis/nodejs-datacatalog/pull/330

**Pull request closed without merging**